### DISCUSSION TITLE
How to create a new ESLint rule for what comes after a property?

### DISCUSSION THREAD

**Discussion by MatanGbr** [eslint/eslint#15455] [Q&A]
Hi all,
I am trying to learn how to write some new ESLint rules for my React projects. I want to start by creating a rule saying that every `value `property should be followed by `trim()` but couldn't figured out how to continue from what I did so far:

```
// Rule: always-trim
'use strict'
module.exports = {
  meta: {
    type: 'suggestion',
    schema: [],
  },
  create(context) {
    return {
    MemberExpression(node) {
        if (node.property.name === 'value') {
          context.report({ node: node.property, message: 'Use trim().' })
        }
      },
    }
  },
}
```
**And I would love to some help.**

**Invalid** should be something like that: `input.value.toString()`
**Valid** should be: `input.value.trim().toString()` (has `trim()` after `value`).

Thanks!

**Comment by nzakas** [comment]
Have you tried using astexplorer.net to see what AST nodes and properties to work with?
